lms/celery.py

```diff
@@ -21,3 +21,42 @@
 # pickle the object when using Windows.
 APP.config_from_object('django.conf:settings')
 APP.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+# Some tasks require settings defined on non-lms workers, or they will crash.
+# This dict defines those tasks, and the worker environment they should be routed towards.
+ALTERNATE_ENV_TASKS = {
+}
+
+
+class Router(object):
```

Comment:
That's what I was wondering - where would I put that definition though? `celery.py` already had some duplicate code, despite being defined under both `lms/` and 'cms/`. Is there a common location that would make sense for this class to live?